packages/playground/vite.config.ts

```diff
@@ -2,14 +2,15 @@ import { visualizer } from "rollup-plugin-visualizer";
 import { definePlaygroundViteConfig } from "./src/build-utils/index.js";
 
 const config = definePlaygroundViteConfig({
-  defaultEmitter: "@typespec/openapi3",
+  defaultEmitter: "@typespec/json-schema",
```

Comment:
Guess we should really have an emitter selector before we merge this

Reply:
Yeah I'm gonna revert all the playground changes before merge, the current thing is just a hack so you can play around with it in the meantime.